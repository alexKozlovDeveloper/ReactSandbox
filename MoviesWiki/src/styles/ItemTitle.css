.title{
    font-family: cursive;
    color: white;
}