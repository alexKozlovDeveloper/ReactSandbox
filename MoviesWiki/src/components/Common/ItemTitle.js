import React, { Component } from "react";

import styles from "../../styles/ItemTitle.css"

const ItemTitle = ({ title }) => {
    return  <div className={styles.title}>
                {title}
            </div>;
}

export default ItemTitle;