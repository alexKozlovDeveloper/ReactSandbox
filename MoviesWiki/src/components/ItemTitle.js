import React, { Component } from "react";

import styles from "../styles/ItemTitle.css"

class ItemTitle extends Component {
    render() {
        return (
            <div className="title">
                <div className={styles.title}>aa {styles.title} aa</div>
                <div>Title {this.props.title}</div>
            </div>
        );
    }
}

export default ItemTitle;