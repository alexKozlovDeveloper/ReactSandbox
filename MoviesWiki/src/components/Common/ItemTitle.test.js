import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import ItemTitle from "./ItemTitle";

configure({adapter: new Adapter()});

test('ItemTitle rendering test', () => {

    var title = "someTitle";

    const output = shallow(<ItemTitle title={title} />);
    expect(shallowToJson(output)).toMatchSnapshot();
 });

