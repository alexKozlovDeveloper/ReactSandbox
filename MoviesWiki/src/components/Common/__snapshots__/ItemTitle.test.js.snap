// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemTitle rendering test 1`] = `
<div
  className="title"
>
  someTitle
</div>
`;
